src/OpenTelemetry.Instrumentation.AspNet/AspNetInstrumentationTracerProviderBuilderExtensions.cs

```diff
@@ -52,7 +52,7 @@ public static TracerProviderBuilder AddAspNetInstrumentation(
                 {
                     return AspNetInstrumentation.Instance.HandleManager.AddTracingHandle();
                 });
-                tracerProviderBuilder.AddSource(TelemetryHttpModule.AspNetSourceName);
+                tracerProviderBuilder.AddSource("OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule");
```

Comment:
Bit of a nit here...

I know this is _technically_ the correct name based on our prior patterns since The `OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule` is the actual package containing the code that produces the span, but it also kind of feels like it exposes an implementation/code organization detail.

I don't feel strongly about this but `OpenTelemetry.Instrumentation.AspNet` "feels" better since it is the package the end user will be more familiar with because it's likely the one they'll take a dependency on.

Approaching this from a different angle, I guess maybe I should be asking someone to remind me why we have this instrumentation separated into two packages? Is it possible for us to merge the two packages?

Reply:
SourceName adjusted